lithops/serverless/backends/aws_lambda/config.py

```diff
@@ -43,6 +49,7 @@
     'Twisted',
     'PyJWT',
     'Pillow',
+    'pandas',
     'redis',
     'pika==0.13.1'
```

Comment:
@aitorarjona Can you add `cloudpickle` `ps-mem` and `tblib` here?

Reply:
@JosepSampe done